### PR TITLE
support funky Microsoft Word XHTML unicode escapes

### PR DESCRIPTION
**Pull request by malsmith**

### CODE REVIEW

**Comment by malsmith** [comment]
These escape sequences are common in MS Word XHTML docs.

**Comment by Alir3z4** [comment]
@malsmith Could you please provide tests as well ?

File: html2text/utils.py [hunk 1]
```diff
@@ -1,8 +1,17 @@
 import sys
 from html2text import config
+import struct 
 
 from html2text.compat import htmlentitydefs
 
+# Based on http://stackoverflow.com/questions/7105874/valueerror-unichr-arg-not-in-range0x10000-narrow-python-build-please-hel
```
Comment (nikolas) [inline html2text/utils.py]: Maybe link to the share link for this question instead: http://stackoverflow.com/q/7105874/173630

**Comment by nikolas** [comment]
Will this fix this error I'm getting?
```
  File "/home/nnyby/src/dmt/dmt/main/migrations/0018_html2text.py", line 13, in populate_comment_src
    comment.comment.decode('unicode-escape'))
  File "/home/nnyby/src/dmt/ve/local/lib/python2.7/site-packages/html2text/__init__.py", line 794, in html2text
    return h.handle(html)
  File "/home/nnyby/src/dmt/ve/local/lib/python2.7/site-packages/html2text/__init__.py", line 122, in handle
    self.feed(data)
  File "/home/nnyby/src/dmt/ve/local/lib/python2.7/site-packages/html2text/__init__.py", line 119, in feed
    HTMLParser.HTMLParser.feed(self, data)
  File "/usr/lib/python2.7/HTMLParser.py", line 117, in feed
    self.goahead(0)
  File "/usr/lib/python2.7/HTMLParser.py", line 191, in goahead
    self.handle_charref(name)
  File "/home/nnyby/src/dmt/ve/local/lib/python2.7/site-packages/html2text/__init__.py", line 165, in handle_charref
    charref = self.charref(c)
  File "/home/nnyby/src/dmt/ve/local/lib/python2.7/site-packages/html2text/__init__.py", line 706, in charref
    return unichr(c)
ValueError: unichr() arg not in range(0x110000) (wide Python build)
```

**Comment by Alir3z4** [comment]
@nikolas Could you try this branch to see if it solve the issue you're facing ?

File: html2text/utils.py (hunk 1 above)
Comment (Alir3z4) [inline html2text/utils.py]: :+1:

**Comment by theSage21** [comment]
@Alir3z4 I tried this branch, the issue is not solved. Failure at the struct module decode.

**Comment by Alir3z4** [comment]
I close this pull-request, the author of the patch @malsmith has not respond regarding to this patch and it comes with conflicts.

**Pull request closed without merging**